backend/src/api/validations/schemaLibrary.validation.ts

```diff
@@ -33,7 +33,7 @@ export const password: z.ZodString = z
 
 export const role: z.ZodEnum<['teacher', 'student']> = z.enum(
     ['teacher', 'student'],
-    errorMessage('Role must be teacher or student'),
+    errorMessage('Role is required'),
```

Comment:
What if user sends an invalid role? I think "Invalid role" should be fine

Reply:
If a user sends an invalid role, it won't work. You are right, i will fix this next PR.